src/db.cpp

```diff
@@ -5542,22 +5542,180 @@ void price_cyber(struct obj_data *obj)
       GET_OBJ_AVAILDAY(obj) = 2;
       break;
     case CYB_SKULL:
-      GET_OBJ_VAL(obj, 1) = 0;
-      GET_CYBERWARE_ESSENCE_COST(obj) = 75;
-      if (GET_OBJ_VAL(obj, 3))
+      GET_OBJ_AVAILDAY(obj) = GET_OBJ_AVAILTN(obj) = GET_CYBERWARE_ESSENCE_COST(obj) = GET_OBJ_COST(obj) = 0;
+      if (IS_SET(GET_OBJ_VAL(obj, 3), SKULL_OBVIOUS)) {
+        GET_OBJ_COST(obj) = 35000;
+        GET_OBJ_AVAILTN(obj) = 6;
+        GET_OBJ_AVAILDAY(obj) = 4;
+        GET_CYBERWARE_ESSENCE_COST(obj) = 75;
+      }
+      if (IS_SET(GET_OBJ_VAL(obj, 3), SKULL_SYNTHETIC)) {
         GET_OBJ_COST(obj) = 55000;
-      else GET_OBJ_COST(obj) = 35000;
-      GET_OBJ_AVAILTN(obj) = 6;
-      GET_OBJ_AVAILDAY(obj) = 4;
+        GET_OBJ_AVAILTN(obj) = 6;
+        GET_OBJ_AVAILDAY(obj) = 4;
+        GET_CYBERWARE_ESSENCE_COST(obj) = 75;
+      }
+      if (IS_SET(GET_OBJ_VAL(obj, 3), SKULL_ARMORMOD1)) {
+        GET_OBJ_COST(obj) += 6500;
+        GET_OBJ_AVAILTN(obj) = MAX(GET_OBJ_AVAILTN(obj), 8);
+        GET_OBJ_AVAILDAY(obj) = MAX(GET_OBJ_AVAILDAY(obj), 14);
+        GET_CYBERWARE_ESSENCE_COST(obj) += 0;
       break;
     case CYB_TORSO:
-      GET_OBJ_VAL(obj, 1) = 0;
-      GET_CYBERWARE_ESSENCE_COST(obj) = 150;
-      if (GET_OBJ_VAL(obj, 3))
+      GET_OBJ_AVAILDAY(obj) = GET_OBJ_AVAILTN(obj) = GET_CYBERWARE_ESSENCE_COST(obj) = GET_OBJ_COST(obj) = 0;
+      if (IS_SET(GET_OBJ_VAL(obj, 3), TORSO_OBVIOUS)) {
+        GET_OBJ_COST(obj) = 90000;
+        GET_OBJ_AVAILTN(obj) = 6;
+        GET_OBJ_AVAILDAY(obj) = 4;
+        GET_CYBERWARE_ESSENCE_COST(obj) = 150;
+      }
+      if (IS_SET(GET_OBJ_VAL(obj, 3), TORSO_SYNTHETIC)) {
         GET_OBJ_COST(obj) = 120000;
-      else GET_OBJ_COST(obj) = 90000;
-      GET_OBJ_AVAILTN(obj) = 6;
-      GET_OBJ_AVAILDAY(obj) = 4;
+        GET_OBJ_AVAILTN(obj) = 6;
+        GET_OBJ_AVAILDAY(obj) = 4;
+        GET_CYBERWARE_ESSENCE_COST(obj) = 150;
+      }
+      if (IS_SET(GET_OBJ_VAL(obj, 3), TORSO_ARMORMOD1)) {
+        GET_OBJ_COST(obj) += 6500;
+        GET_OBJ_AVAILTN(obj) = MAX(GET_OBJ_AVAILTN(obj), 8);
+        GET_OBJ_AVAILDAY(obj) = MAX(GET_OBJ_AVAILDAY(obj), 14);
+        GET_CYBERWARE_ESSENCE_COST(obj) += 0;
+      }
+      if (IS_SET(GET_OBJ_VAL(obj, 3), TORSO_ARMORMOD2)) {
+        GET_OBJ_COST(obj) += 13000;
+        GET_OBJ_AVAILTN(obj) = MAX(GET_OBJ_AVAILTN(obj), 8);
+        GET_OBJ_AVAILDAY(obj) = MAX(GET_OBJ_AVAILDAY(obj), 14);
+        GET_CYBERWARE_ESSENCE_COST(obj) += 0;
```

Comment:
If none of these mods increase the essence cost, then I would remove this, as it's a non-operation. If you're trying to make it clear that it doesn't increase the armor cost, just add a comment to that effect.

Reply:
Some of the extra stuff I wanted to potentially add down the line will cost an essence increase, I just included it to be safe, I can remove em for any that don't require additional essence costs though.